src/content/nft/index.md

```diff
@@ -0,0 +1,387 @@
+
+As we mentioned, this is an emerging space. NFTs, DAOs, fractionalised tokens are all developing at different paces. But all their infrastructure exists and can work together easily because they all speak the same language: Ethereum. So watch this space.
+
+[More on DAOs](/dao/)
+
+<Divider />
+
+## Ethereum and NFTs {#ethereum-and-nfts}
+
+Ethereum makes it possible for NFTs to work for a number of reasons:
+
+- Transaction history and token metadata is publicly verifiable – it's simple to prove ownership history.
+- Once a transaction is confirmed, it's nearly impossible to manipulate that data to "steal" ownership.
+- Trading NFTs can happen peer-to-peer without needing platforms that can take large cuts as compensation.
+- All Ethereum products share the same "backend". Put another way, all Ethereum products can easily understand each other – this makes NFTs portable across products. You can buy an NFT on one product and sell it on another easily. As a creator you can list your NFTs on multiple products at the same time – every product will have the most up-to-date ownership information.
+- Ethereum never goes down, meaning your tokens will always be available to sell.
+
+## The environmental impact of NFTs {#environmental-impact-nfts}
+
+NFTs are growing in popularity which means they're also coming under increased scrutiny – especially over their carbon footprint.
+
+To clarify a few things:
+
+- By pressing the button to create or buy an NFT you are not directly releasing carbon into the atmosphere.
+- NFTs aren't directly increasing the carbon footprint of Ethereum.
+
+Here's where things are going to get a little technical, but bear with us...
+
+### Don't blame it on the NFTs
+
+The whole NFT ecosystem works because Ethereum is decentralized and tamper-proof.
+
+Decentralized meaning you and everyone else can verify you own something without trusting or granting custody to a centralized third party who can impose their own rules at will. It also means your NFT is portable across many different products and markets.
+
+Tamper-proof meaning no one can copy/paste your NFT or steal it.
+
+These qualities of Ethereum makes digitally owning unique items and monetising your content fairly possible.
+
+Blockchains like Bitcoin and Ethereum are energy intensive right now because it takes a lot of energy to preserve these qualities. If it was easy to rewrite Ethereum's history to steal NFTs or cryptocurrency, the system collapses.
+
+<!-- [Need: how do centralized systems do this and why is it worse?]
+ -->
+
+#### The work in minting an NFT
+
+When you mint an NFT, a few things have to happen:
+
+- it needs to be confirmed as an asset on the blockchain.
+- the owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".
+- the transactions that confirm the above need to be added to a block and "immortalised" on the chain.
```

Comment:
These first 3 points seem to all say the same thing, though perhaps that helps to clarify it 🤷

Reply:
they're not sequential. I believe it's necessary to paint the entire picture of what's happening.